content/cloud/cloud-command-reference.md

```diff
@@ -587,6 +731,59 @@ OPTIONS:
 
 {{ blockEnd }}
 
+{{ startTab "v0.4.0"}}
+
+Spin compatibility: `>= v1.3`
+
+<!-- @selectiveCpy -->
+```console
+$ spin cloud deploy --help
+
+Package and upload an application to the Fermyon Cloud
+
+USAGE:
+    spin cloud deploy [OPTIONS]
+
+OPTIONS:
+        --build
+            Specifies to perform `spin build` before deploying the application [env:
+            SPIN_ALWAYS_BUILD=]
+
+        --buildinfo <BUILDINFO>
+            Build metadata to append to the oci tag
+
+        --environment-name <environment-name>
+            Deploy to the Fermyon instance saved under the specified name. If omitted, Spin deploys
+            to the default unnamed instance [env: FERMYON_DEPLOYMENT_ENVIRONMENT=]
+
+    -f, --from <APP_MANIFEST_FILE>
```

Comment:
@karthik2804 While we're here adding the 0.4.0 tab for this deploy command, can you regenerate using the canary version of the plugin?  This flag's text has changed slightly in 0.4.0 to indicate support for oci references:
```
    -f, --from <APPLICATION>
            The application to deploy. This may be a manifest (spin.toml) file, a directory
            containing a spin.toml file, or a remote registry reference. If omitted, it defaults to
            "spin.toml"
```